pyproject.toml

```diff
@@ -216,7 +219,7 @@ formatting = [
 ]
 
 all = [
-  "farm-haystack[inference,docstores,audio,crawler,preprocessing,file-conversion,pdf,ocr,ray,onnx,beir,metrics]",
+  "farm-haystack[inference,docstores,audio,crawler,preprocessing,file-conversion,pdf,ocr,ray,onnx,beir,metrics,aws]",
 ]
 all-gpu = [
```

Comment:
For consistency I'd add `aws` in the gpu deps too, wdyt?

Reply:
yes please, for sure